### PR TITLE
Hongbaoshila Machine Learning: Translation

### PR DESCRIPTION
**Pull request by hongbaoshila**

### CODE REVIEW

**Review by metafly — APPROVED** [review]

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -50,7 +50,12 @@ function showMessageFormIfViewingSelf() {
 
 /** Fetches messages and add them to the page. */
 function fetchMessages() {
-  const url = '/messages?user=' + parameterUsername;
+  //const url = '/messages?user=' + parameterUsername;
```
Comment (SeanCampbell) [inline src/main/webapp/js/user-page-loader.js]: Nitpick: Can you remove this line?

**Review by SeanCampbell — APPROVED** [review]
Nice work! This looks good to me, except one nitpicky comment.

Is this the full working translation, or just the first part?

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (hongbaoshila) [reply src/main/webapp/js/user-page-loader.js]: Sure

**Comment by hongbaoshila** [comment]
> Nice work! This looks good to me, except one nitpicky comment.
> 
> Is this the full working translation, or just the first part?

This is the full part expect the "your turn" part. I want to move on to the next project because I don't think we need this functionality when we are doing only 1 university instead of universities around the world. If we find out we need translation in the future, I can come back to it.

**Pull request merged**